dhcptest.d

```diff
@@ -501,7 +524,10 @@ bool receivePackets(Socket socket, bool delegate(DHCPPacket, Address) handler, D
 				return true;
 		}
 		catch (Exception e)
+		{
 			stderr.writefln("Error while parsing packet [%(%02X %)]: %s", receivedData, e.toString());
+			end = Clock.currTime();
```

Comment:
> That code is to ensure that the while loop would immediately exit on the next go-around.

OK, so the statement is synonymous with `break;`?

> which if none came meant the program hung until "timeout".

OK, but this addition also changes the behavior when actually receiving a malformed DHCP packet. So, if you're using `--wait`, the first malformed DHCP packet makes the program exit early instead of the  expected behavior of printing all received values for the valid packets received within the timeout.